src/System.Reflection.TypeExtensions/tests/MethodInfoTests.cs

```diff
@@ -13,7 +13,7 @@ public class MethodInfoTests
         [Theory]
         [InlineData(typeof(MI_NonGenericClass), nameof(MI_NonGenericClass.PublicMethod))]
         [InlineData(typeof(MI_NonGenericClass), nameof(MI_NonGenericClass.PublicStaticMethod))]
-        [InlineData(typeof(string), nameof(string.IsNullOrEmpty))]
+        [InlineData(typeof(StringImpersonator), nameof(StringImpersonator.IsNullOrEmpty))]
```

Comment:
There was another test that was using System.String and Int32 as test data and probing for all kinds of members on it. Simplest way to fix the test was to give it what it wanted to see.